common/MultiTarget/Uno.props

```diff
@@ -0,0 +1,34 @@
+<Project>
```

Comment:
Same here, was this not a move?

Reply:
This file was one directory up, in the `./common` folder. It pertains to targeting the different platforms, so it was moved here.